drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -82,45 +82,58 @@
     </div>
 </script>
 
-<script id="zone-dropdown-tpl" type="text/html">
-    <option value="{{ uid }}" {{ selected }}>{{ title }}</option>
+<script id="zone-checkbox-tpl" type="text/html">
+    <div class="row">
+        <label>
+            <input type="checkbox"
+                   value="{{ zoneUid }}"
+                   class="zone-checkbox"
+                   {{ checked }} />
+            {{ title }}
+        </label>
+    </div>
 </script>
 
 <script id="item-input-tpl" type="text/html">
     <div class="item">
         <div class="row">
-            <label for="item-{{id}}-text">{{i18n "Text"}}</label>
+            <label for="item-{{id}}-text" class="h3 zone-checkbox-label">{{i18n "Text"}}</label>
             <input type="text"
                    id="item-{{id}}-text"
                    class="item-text"
                    value="{{ displayName }}" />
-            <label for="item-{{id}}-zone">{{i18n "Zone"}}</label>
-            <select id="item-{{id}}-zone"
-                    class="zone-select">{{ dropdown }}</select>
             <a href="#" class="remove-item hidden">
                 <div class="icon remove"></div>
             </a>
         </div>
         <div class="row">
-            <label for="item-{{id}}-image-url">{{i18n "Image URL (alternative to the text)"}}</label>
+            <fieldset id="item-{{ id }}-zone">
```

Comment:
@haikuginger `item-{{ id }}-zone` doesn't seem to be referenced anywhere. If that's the case you can remove it.